plugins/node/opentelemetry-instrumentation-pino/test/pino.test.ts

```diff
@@ -303,4 +307,39 @@ describe('PinoInstrumentation', () => {
       });
     });
   });
+
+  it('should work with ESM usage', async function () {
+    if (semver.lt(pino.version, '6.0.0')) {
+      // Pino v5 did not support named exports.
```

Comment:
Thank you for commenting this!
Do we need another branch in this logic for default import for Pino v5 then?

Reply:
You are right, though, that I could split the ESM testing into two separate tests: one that does default import and one that does named imports.  I started with both of them together... then discovered the pino@5 limitation and arguably got lazy. :)

Reply:
Done in commit b10ec11.

Reply:
Noting for posterity:

> I don't know if OTel has a specific policy for support of instrumentations of really old package versions.

As we chatted about in SIG meeting, we don't have a specific policy in place right now for package versions. For now we're going with: If it's a relatively small lift to test older versions, let's keep doing it. Once it starts causing more significant pain, we should revisit and handle those on a case-by-case basis. It's difficult to have a standard policy across all packages since they are all so different, but we'll see how things shape up in the future.